buildconfig/stubs/pygame/__init__.pyi

```diff
@@ -633,4 +632,3 @@ from .constants import (
     WINDOWSIZECHANGED as WINDOWSIZECHANGED,
     WINDOWTAKEFOCUS as WINDOWTAKEFOCUS,
 )
-
```

Comment:
> minor nitpick, It'd be better if this file is not modified by hand because it is generated by a script. So the next time someone runs the script, it'd generate diffs again when these linebreaks are re-added

updated generator to generate black conform files

Reply:
That's a good improvement!